.travis.yml

```diff
@@ -85,9 +85,8 @@ after_success:
 deploy:
   # Deploy Docs to Github Pages when tagged as release
   provider: pages
-  strategy: git
-  cleanup: false
-  token: $GITHUB_TOKEN
+  skip_cleanup: true
+  github_token: $GITHUB_TOKEN
```

Comment:
same for `GitHub_token`

Reply:
Same for this one, it seems the github token wasn't working in the last release: : https://travis-ci.org/github/rakutentech/ios-miniapp/builds/715699238